opm/utility/ECLGraph.cpp

```diff
@@ -274,6 +279,19 @@ namespace {
                 std::vector<T>
                 scatterToGlobal(const std::vector<T>& x) const;
 
+                /// Map input vector to active grid cells.
+                ///
+                /// \param[in] x Input vector, defined on the explicitly
+                ///              active cells, all global cells or some
+                ///              other subset (e.g., all non-neighbouring
+                ///              connections).
+                ///
+                /// \return Input vector mapped to active cells or unchanged
```

Comment:
> The doc needs to be clear about the `ACTNUM` case, then.

I'm rewriting this now.

> Since it is a little hard to explain, perhaps it would be better to have separate methods for gathering global -> active and nonzero-actnum -> active?

I'd really prefer not, because I don't want to expose the case switching to callers.